File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.js
function solution(a, b) {
    var answer = a.toString()
    var answer1 = b.toString()
    var answer2 = answer + answer1
    
    var answer3 = 2 * a * b
    
    if(answer2 >= answer3) return Number(answer2)
    else return answer3;
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        String str = Integer.toString(a) + Integer.toString(b);
        int value = Integer.parseInt(str);
        if(value > 2 * a * b) {
            answer = value;
        }
        else {
            answer = 2 * a * b;
        }
        return answer;
    }
}